### ISSUE TITLE
Dockerize

### ISSUE THREAD

**Issue by jpadilla** [jpadilla/notaso#72]
Llevo días debatiendo si quiero Dockerize este proyecto con la posibilidad de tener un setup para deploy en Digital Ocean o en algún otro provider. Mayormente no quiero pagar $20 por solo tener SSL enabled en Heroku.

Maybe es un buen ejercicio para el Docker Meetup.

**Comment by eluciano11** [comment]
mmmm yo nunca he usado docker pero aprendo.

**Comment by jpadilla** [comment]
https://registry.hub.docker.com/u/dockerfile/nginx/
https://registry.hub.docker.com/_/postgres/

**Comment by jpadilla** [comment]
Terraform es un overkill for this.

Trying out tutum, just for the heck of it. Already have automatic builds in Docker Hub setup.

Might check out Cloudflare to deal with HTTPS.

**Comment by jpadilla** [comment]
Want to check out https://github.com/phusion/passenger-docker.

Setup idea until now:
Container with app and its requirements running with gunicorn and nginx
Private container with SSL and nginx configuration as a volume to the container above.

**Comment by jpadilla** [comment]
nginx container(notaso-web) + gunicorn container(notaso-app) is mostly working.

Next steps: Look into settings nginx + gunicorn(notaso-web + notaso-app) in one container, passenger-docker might work well for this setup, and attach notaso-data container for SSL certs.

**Comment by Jpadilla1** [comment]
@jpadilla 
I think we should use [www.fig.sh](www.fig.sh) to setup the containers for a simple setup, linking up containers and easily scale any container if needed.

**Comment by jpadilla** [comment]
@Jpadilla1 sí, no estaría mal. Still need to figure out the setup without fig first.

**Comment by Jpadilla1** [comment]
I've been playing around with docker for a few weeks now and finally got the hang of it. I'll investigate the setup with fig probably this weekend.

**Comment by jpadilla** [comment]
@Jpadilla1 sería bueno comparar notas or meetup later para meterle un rato. Voy a ver si hoy logro continuar.

**Comment by jpadilla** [comment]
http://blog.commando.io/the-perfect-nginx-ssl-configuration/

**Comment by jpadilla** [comment]
Will probably work on this, but focusing on development, docker + compose

**Comment by Jpadilla1** [comment]
Happy to help out, been using docker lately
